src/test/java/seedu/address/logic/parser/SortCommandParserTest.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.sort.SortCommand;
+import seedu.address.logic.commands.sort.SortInternshipsCommand;
+import seedu.address.logic.commands.sort.SortProjectsCommand;
+import seedu.address.model.item.Item;
+
+public class SortCommandParserTest {
+
+    private SortCommandParser parser = new SortCommandParser();
+
+    // NOTE: All userInput must be preceded by a space.
```

Comment:
Is this for consistency of Strings in program or does our commands not work if there is no space?

Reply:
It's for all commands.